### PR TITLE
Webhooks for team event-types

### LINKED ISSUE #5935: [CAL-564] Add back the ability to create webhooks on an event type
A temporary fix before we allow webhook creation on an org level is to add back the ability to create them per event type. We used to have this logic before.

Related: [https://github.com/calcom/cal.com/issues/5735](https://github.com/calcom/cal.com/issues/5735)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-564](https://linear.app/calcom/issue/CAL-564/add-back-the-ability-to-create-webhooks-on-an-event-type)</sub>

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Adds back the ability to create webhooks for team event types (PR #1889 preciously implemented that). 

Fixes #5935

And a temporary fix for: #5735

 Loom Video: https://www.loom.com/share/524a30298e6c4549aa3de6739d186a3f

**Environment**: Staging(main branch)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

- Go to editing view of a team event type
- Go to webhooks 
- Test creating, editing and deleting a webhook

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/features/webhooks/pages/webhook-new-view.tsx [hunk 1]
```diff
@@ -34,12 +34,14 @@ const NewWebhookView = () => {
...
-  const subscriberUrlReserved = (subscriberUrl: string, id: string): boolean => {
-    return !!webhooks?.find((webhook) => webhook.subscriberUrl === subscriberUrl && webhook.id !== id);
+  const subscriberUrlReserved = (subscriberUrl: string, id?: string): boolean => {
+    return !!webhooks?.find(
+      (webhook) => webhook.subscriberUrl === subscriberUrl && (!id || webhook.id !== id)
+    );
   };
 
   const onCreateWebhook = async (values: WebhookFormSubmitData) => {
-    if (values.id && subscriberUrlReserved(values.subscriberUrl, values.id)) {
```
Comment (CarinaWolli) [inline packages/features/webhooks/pages/webhook-new-view.tsx:42]: values.id is always undefined when creating webhook

**Review by zomars — APPROVED** [review]
LGTM

**Comment by gzim-joinfella** [comment]
hey @CarinaWolli 

I see that this is merged. any idea when it will be released? thanks!

**Comment by CarinaWolli** [comment]
@gzim-joinfella It should be released later today. You should then be able to create webhooks for your team event types

**Comment by aasargsian** [comment]
@CarinaWolli thanks a lot, we'll test this feature! I was wondering though if there are any plans to implement webhooks on the organisation level? If yes, I would like to watch the thread to be notified once the feature is available :)

For more context: We set up webhooks separately for each user and it is working. But this is challenging to set up and maintain with bigger and growing teams. It would be much easier to have 1 centralised webhook for the organisation.
Just as an example, in Calendly we would use 1 webhook for the entire account including all teams and users

**Comment by CarinaWolli** [comment]
@aasargsian Yes, we are working on allowing to create webhooks on an organisational level. @Jaibles do we have an open issue for that?

**Pull request merged**